### PR TITLE
Config cleanup

### PR DESCRIPTION
**Pull request by cpheps**
#### Description of the changes

- Changed the `.template` back to a `.sample`
- Added verbose descriptions to the config file to help alleviate configuration issues
- Fixed metric spelling errors in the spec.csv

### Author

- [X] add a risk label after carefully considering the "blast radius" of your changes
- [X] describe the _intent_ of your changes in the description. don't just rewrite your code in prose
- [X] assign at least one reviewer

### Reviewer

- [ ] review code for readability
- [ ] verify that high risk behavior changes are well tested
- [ ] check license for any new external dependency
- [ ] ask questions about anything that isn't clear and obvious
- [ ] approve the PR when you consider it's good to merge

### CODE REVIEW

File: kafka-config.yml.sample [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    arguments:
+
+      # More information about the zookeeper connection can be found above
+      zookeeper_hosts: <JSON Array of Zookeeper Hosts of the form '[{"host": "localhost", "port": 2181}]'>
+      zookeeper_auth_scheme: <Auth scheme for Zookeeper>
+      zookeeper_auth_secret: <Auth string for Zookeeper>
+      zookeeper_path: <Root path of Kafka nodes in Zookeeper>
+
+      # Below are the fields used to fine tune/toggle topic inventory collection.
+      # In order to collect topics the "topic_mode" field must be set to "All" or "List". If the field is set to "All"
```
Comment (davidbrota) [inline kafka-config.yml.sample]: I think something is missing on the last phrase.

Comment (cpheps) [reply kafka-config.yml.sample]: Not sure I follow, it continues on the next line. We broke it up for readability. The whole line should read 

```
If the field is set to "All" a Zookeeper connection is required, at least the "zookeeper_hosts" field is required, as topics are looked up via Zookeeper.
```

Comment (davidbrota) [reply kafka-config.yml.sample]: Ok, but next line is empty. Maybe are you referring to line 64?

Comment (cpheps) [reply kafka-config.yml.sample]: Might be missing something due to how GitHub show comments. So line 64 is a continuation of the end of line 63. Line 65 is blank just to signal a new paragraph starting on line 66.

Comment (davidbrota) [reply kafka-config.yml.sample]: Line 63..66 are fine, I'm talking about current line 92 that ends with _If the field is set to "All"_ and the next line 93 is empty. Sorry for the misunderstanding.

Comment (cpheps) [reply kafka-config.yml.sample]: Yup totally didn't notice that...sorry. Fixed it!

**Pull request merged**